layouts/partials/fragments/single-member.html

```diff
@@ -0,0 +1,134 @@
+{{ $single := .self.Params }}
+{{ $name := .self.Name }}
+{{ $name_t := strings.TrimSuffix ".md" $name }}
+{{ $member := .resources.Match (printf "%s/*" $name_t) }}
+{{ "<!-- Single Member -->" | safeHTML }}
+{{ $bg := $single.background | default "light"}}
+
+<section {{ with $single.self_link }}id="{{ . }}"{{ end }}>
+  <div class="overlay container-fluid
+    {{- printf " bg-%s" $bg -}}
+  ">
+    <div class="container d-flex flex-column py-3">
+      {{ with $single.title }}
+        <div class="row">
+          <div class="col text-center
+            {{- if or (eq $bg "white") (eq $bg "light") (eq $bg "secondary") (eq $bg "primary") -}}
+              {{- printf " text-%s" "dark" -}}
+            {{- else -}}
+              {{- printf " text-%s" "light" -}}
+            {{- end -}}
+          ">
+            <h2>
+              {{- . | markdownify -}}
+            </h2>
+          </div>
+        </div>
+      {{ end }}
+      <!-- {{ with $single.subtitle }}
+        <div class="row">
+          <div class="col text-center pt-2 pb-3
+            {{- if or (eq $bg "secondary") (eq $bg "primary") -}}
+              {{- printf " text-%s" "dark" -}}
+            {{- else -}}
+              {{- printf " text-muted text-%s" "secondary" -}}
+            {{- end -}}
+          ">
+            <h5>
+              {{- . | markdownify -}}
+            </h5>
+          </div>
+        </div>
+      {{ end }} -->
```

Comment:
I guess this is leftover. Or is it needed?

Reply:
No it's not. I'll remove it.